app/config.py

```diff
@@ -41,46 +41,46 @@ class Settings(BaseSettings):
     host: str = '0.0.0.0'
     env: str = ''
     VERSION: str = '0.2.3'
-    namespace: str = ''
+    namespace: str = 'DATASET'
 
     DATASET_FILE_FOLDER: str = 'data'
-    DATASET_SCHEMA_FOLDER: str = 'schema'
 
     DATASET_CODE_REGEX: str = '^[a-z0-9]{3,32}$'
 
     # disk mounts
     ROOT_PATH: str
 
-    CORE_ZONE_LABEL: str
-    GREEN_ZONE_LABEL: str
+    CORE_ZONE_LABEL: str = 'core'
 
     # minio
-    MINIO_OPENID_CLIENT: str
     MINIO_ENDPOINT: str
     MINIO_HTTPS: bool = False
-    KEYCLOAK_URL: str
     MINIO_ACCESS_KEY: str
     MINIO_SECRET_KEY: str
+    KEYCLOAK_URL: str
     KEYCLOAK_MINIO_SECRET: str
 
+    # External services
     QUEUE_SERVICE: str
     CATALOGUING_SERVICE: str
```

Comment:
This one and related occurrences should be renamed into Lineage Service.
```suggestion
    LINEAGE_SERVICE: str
```